testdata/scripts/loader_files.txt

```diff
@@ -0,0 +1,15 @@
+env HOME=$WORK/home
+
+gunk format p1/p1.gunk
```

Comment:
Do we also want to compare the .gunk files to the .gunk.golden files?

Reply:
Agh, I wrote the first version of this test, and forgot to actually finish it. It's missing at least a handful of lines.